src/generic/parser.h

```diff
@@ -49,7 +49,7 @@ struct string {
 typedef struct mnemonic mnemonic_t;
 struct mnemonic {
   struct {
-    char data[16];
+    char data[24];
```

Comment:
This array is capped at 16 bytes because it's used for comparison using SIMD `cmpeq`. Better remove usage of `mnemonic` in SvcParam definitions.

Reply:
Resolved in commit 654de1e

Reply:
Not really. The array is capped because the comparison uses exactly 16 bytes when looking up type names and the compiler is meant to complain if someone tries to fit something larger because assumptions are broken otherwise. Now that some SVCB parameter requires more, and with the SvcParamKey no being a mnemonic technically, it's cleaner to fix the issue by defining something like name directly in `svc_param_info_t` or something.

Reply:
I was thinking like a separate `svc_param_key_t` so we can optimize resolving the name later on (now that the presentation format for `SVCB` is use more and more), but I'll leave that to you.

Reply:
Nevermind, it was me not looking at the commit (duh). Looks good :+1: